sematic/runners/local_runner.py

```diff
@@ -282,14 +284,9 @@ def _cancel(data):
 
             logger.warning("Received cancelation event")
 
-            root_run = self._get_run(self._root_future.id)
-            if root_run.future_state != FutureState.CANCELED.value:
-                raise RuntimeError("Cancelation was not effective.")
-
-            # If we are here, the cancelation was applied successfully server-side
```

Comment:
Please preserve this specific comment line in order for the intent of this code to be completely clear.

Reply:
done